src/main/java/com/google/sps/data/Mentor.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Mentor extends UserAccount {
+
+  public static final String ENTITY_TYPE = "Mentor";
+
+  private String dateOfBirth;
```

Comment:
Do you really want the DOB to be a String? What if you want to display the mentor's age, or maybe show a cake next to their name on their bday? You'd have to parse it every time into a date object.